src/main/scala/org/ergoplatform/modifiers/history/Header.scala

```diff
@@ -37,9 +37,8 @@ case class Header(version: Version,
 
   override lazy val id: ModifierId = ModifierId @@ powHash
 
-  //todo: why SHA256?
-  //todo: tolsi: check this
   lazy val powHash: Digest32 = {
+    // implementations of pow hash similar to ZCash.
```

Comment:
An implementation of a PoW function which is similar to one used in ZCash.

Reply:
fixed